tests/run-unittests.mjs

```diff
@@ -0,0 +1,57 @@
+#! /usr/bin/env node
+/* eslint-disable-next-line  no-unused-vars */
```

Comment:
nit: it could be good to add a comment explaining a few things about the tests, such as "tests are run in the browser by the `index.html` file. Individual tests are defined in benchmark-runner-tests.html", things like that.